<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSGBoat</class>
 <widget class="QMainWindow" name="OSGBoat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,1" columnminimumwidth="5,1">
      <item row="0" column="0">
       <layout class="QGridLayout" name="boxlayout"/>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLabel" name="label_MoveState">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Null</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_MoveStateTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>MoveState:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_SternWarningX">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Danger</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_HeadWarningXTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HeadWarningX:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_HeadWarningYTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>HeadWarningY:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_SternWarningXTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SternWarningX:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_HeadWarningX">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Danger</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_HeadWarningY">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Danger</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_Time">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>00:00:0000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_Coor">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>BoatRate:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_accommodationTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ParkingSpace:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_CoordinateTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Coordinate:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_RadioTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Radio:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_TimeTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Time:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLabel" name="label_Accommodation">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_Distance2title">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Distance2:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_NumTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>BoatID:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_Coordinate">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>(0.00,0.00)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_LinkState">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DisConnection</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QPushButton" name="pushButton_vertical">
          <property name="text">
           <string> Vertical View</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QPushButton" name="pushButton_front">
          <property name="text">
           <string>Front View</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_DistanceTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Distance:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_ID">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_Distance1title">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Distance1:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_GPS2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>False</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_Distance">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.00(m)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_GPS1title">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GPS1:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_Rate">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_GPS1">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>False</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QPushButton" name="pushButton_closeLink">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_LinkStateTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>LinkState:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_Distance2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.0,0.0</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QPushButton" name="pushButton_lateral">
          <property name="text">
           <string>Lateral View</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_GPS2title">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GPS2:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="label_Distance1">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.0,0.0</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="label_Radio">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_SternWarningYTitle">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SternWarningY:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_SternWarningY">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Danger</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChange">
    <property name="title">
     <string>Link</string>
    </property>
    <addaction name="actionChangeView_LinkBoat"/>
   </widget>
   <addaction name="menuChange"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChangeView_LinkBoat">
   <property name="text">
    <string>LinkBoat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
